spec/requests/internal/chat_channel_spec.rb

```diff
@@ -21,4 +21,14 @@
       expect(ChatChannel.last.pending_users).to include(user)
     end
   end
+
+  describe "DELETE /internal/chat_channels/:id/remove_user" do
+    it "removes the user from the chat channel" do
+      delete "/internal/chat_channels/#{user.id}/remove_user",
+             params: { chat_channel: { usernames_string: user.username.to_s } }
```

Comment:
Noting to update this param to singular `username_string`.

Also `user.username` always returns a string so you wont need `.to_s`.